spec/models/mef_spec.rb

```diff
@@ -11,4 +11,43 @@
   it { is_expected.to validate_presence_of(:mefstat11) }
   it { is_expected.to validate_presence_of(:label) }
   it { is_expected.to validate_presence_of(:short) }
+
+  describe "bop_code" do
+    context "when the mef.ministry is no MENJ" do
+      let(:mef) { build(:mef, ministry: "masa") }
+      let(:establishment) { create(:establishment) }
+
+      it "returns the ministry" do
+        expect(mef.bop_code(establishment)).to eq "masa"
+      end
+    end
+
+    context "when the mef.ministry is MENJ" do
+      let(:mef) { build(:mef, ministry: "menj") }
+
+      context "when the establishment status contract is 'no contract'" do
```

Comment:
is public ?

Reply:
ben non en soit ce statut 99 ce qu'il dit c'est "sans objet"

Reply:
sans objet, pas sans contrat 😄 bon enfin comme tu veux

Reply:
oui ok je peux préciser ça mieux